sig/sig-dataScience/charter.md

```diff
@@ -0,0 +1,44 @@
+# SIG Data Science Charter
+
+This charter adheres to the conventions described in the [Open Services Governance document](https://github.com/open-services-group/community/blob/main/governance.md) and uses the Roles and Organization Management outlined in [sig-governance](https://github.com/open-services-group/community/blob/main/committee-steering/governance/sig-governance.md).
+
+## Scope
+
+SIG Data Science is responsible for the development, exploration and communication of all data science and machine learning initiatives within the open services group. This includes cloud native machine learning tools and best practices, as well as AIOps tools to enhance the services run by the open services group.
+
+
+### In scope
+
+#### Code, Binaries and Services
+
+* Code and Notebooks for data management, analysis, model training, model performance and model inference for sub-projects defined by this SIG.
+* Deployments and maintenance of model services and pipelines
+* Development and maintenance of dashboards
+* Model service monitoring
+* Publish and promotion of videos, blogs, talks and tutorials on the tools and best practices developed by sub-projects.
+
+
+
+#### Cross-cutting and Externally Facing Processes
+
+* Support external teams onboarding data science work to any open services managed clusters.
+* Work with other SIG's to ensure data science use-cases are correctly met by new tools and services where appropriate.
+* Work with other SIG’s to ensure data collection and reporting is ongoing.
+
+
+### Out of scope
+
+* Not responsible for the installation and availability of data science services maintained by the Open Services Org
```

Comment:
is this "Open Services Org" or "Operate First Initiative" ?

Reply:
good question :) this charter is for the Open Services Org, so wanted to keep the language consistent. Also, does it make sense to consider the Open Services Org as managing the Operate First Initiative? If so, then I want to keep this as high level and broad as possible, so I think "open services org" makes the most sense, but in reality it is both. WDYT?